atlassian_jwt_auth/frameworks/common/backend.py

```diff
@@ -38,7 +42,10 @@ class Backend():
         'ASAP_VALID_ISSUERS': None,
 
         # Enforce that the ASAP subject must match the issuer
-        'ASAP_SUBJECT_SHOULD_MATCH_ISSUER': None
+        'ASAP_SUBJECT_SHOULD_MATCH_ISSUER': None,
+
+        # Enforce that tokens have a unique JTI
+        'ASAP_CHECK_JTI_UNIQUENESS': None,
```

Comment:
I am thinking that for historical reasons it makes more sense to default to the JTI not being checked by default.

Reply:
I think that's fine; the spec calls out that if people are enforcing it, they need to document it — thus this is the safe default:

> The resource server MAY reject a token if the token nonce (“jti”) has been previously seen by the resource server in another request. If the resource server decides to implement duplicate detection, it MUST explicitly document that behaviour.